prow/update-jobs/Makefile

```diff
@@ -1,36 +1,67 @@
 SHELL := /bin/bash
 
+# OCI image artifact variables
+OCI_AWS_ECR_REGION ?= eu-west-1
+OCI_AWS_ECR_ACCOUNT ?= 292999226676
+OCI_AWS_ECR_NAME := dkr.ecr.$(OCI_AWS_ECR_REGION).amazonaws.com
+OCI_AWS_ECR_REGISTRY := $(OCI_AWS_ECR_ACCOUNT).$(OCI_AWS_ECR_NAME)
+OCI_IMG_REPOSITORY := test-infra/update-jobs
+OCI_IMG_NAME := $(OCI_AWS_ECR_REGISTRY)/$(OCI_IMG_REPOSITORY)
+OCI_IMG_TAG ?= latest
+
 VERBOSE ?= 0
 ifeq ($(VERBOSE), 0)
 # Make is verbose in Linux: let's make it silent.
 MAKEFLAGS += --silent
 endif
 
 PROJECTNAME := update-jobs
-BASE := $(shell pwd)
-OUTPUT := $(BASE)/bin
+OUTPUT_BASE ?= $(shell pwd)
+OUTPUT_DIR := bin
+OUTPUT_PATH := $(OUTPUT_BASE)/$(OUTPUT_DIR)
 
 # Go variables
 GOFILES := $(wildcard *.go)
 
 # Redirect error output to a file, so we can show it in development mode.
 STDERR := "/tmp/.$(PROJECTNAME)-stderr.txt"
 
-$(OUTPUT)/$(PROJECTNAME): $(GOFILES)
+$(OUTPUT_PATH)/$(PROJECTNAME): $(GOFILES)
 
-build: $(OUTPUT)/$(PROJECTNAME)
-	@echo "  >  Building binary..."
+build: $(OUTPUT_PATH)/$(PROJECTNAME)
+	@echo "  >  Building binary... "
 	touch $(STDERR)
 	rm $(STDERR)
-	mkdir -p $(OUTPUT)
-	go build -o "$(OUTPUT)/$(PROJECTNAME)" $(GOFILES) 2> $(STDERR)
-	cat $(STDERR) | sed -e '1s/.*/\nError:\n/'  | sed 's/make\[.*/ /' | sed "/^/s/^/     /" 1>&2
+	mkdir -p $(OUTPUT_PATH)
+	go build -o "$(OUTPUT_PATH)/$(PROJECTNAME)" $(GOFILES)
+	@echo "Compiled in: $(OUTPUT_PATH)/$(PROJECTNAME)"
```

Comment:
The `cat ...` here is needed to show the errors that occur during the build

Reply:
Shouldn't we be able to read it?  Anyway I can revert it  :)

Reply:
In the meantime I reverted it

Reply:
Agree let's edit the existing Dockerfile in `images/update-jobs`

Reply:
Agree @leodido @jonahjon I got the point, let's keep images and prow sources separate :)

Reply:
I noticed that also packages downloading info are written to stderr